<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EntryEditor</class>
 <widget class="QWidget" name="EntryEditor">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>780</width>
    <height>430</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">/*background-color: rgb(42, 42, 42);*/

QLabel
{
	color: rgb(216, 216, 216);
}

QPushButton 
{	
	color: rgb(216, 216, 216);
	background-color: rgb(0, 0, 0);
	border-radius: 10px;
}

QPushButton:hover 
{
	background-color: rgb(35, 11, 63);
}

QPushButton:disabled
{
	color: rgb(73, 73, 73);
}

QPushButton:pressed 
{
	background-color: rgb(82, 26, 149);
}

QTableWidget
{
	background-color: rgb(0, 0, 0);
}

QTableWidget::item::selected
{
	background-color: rgb(0, 58, 201);
}

QHeaderView::section
{
	background-color: rgb(20, 6, 36);
}

QHeaderView
{
	background-color: rgb(30, 30, 30);
}

QLineEdit
{
	color: rgb(216, 216, 216);
	background-color: rgb(0, 0, 0);
	padding-left:10px;
}

QTextEdit
{
	color: rgb(216, 216, 216);
	background-color: rgb(0, 0, 0);
	padding-left:25px;
}

QComboBox
{
	color: rgb(216, 216, 216);
	background-color: rgb(0, 0, 0);
}

QComboBox QAbstractItemView {
 	border: 1px solid grey;
 	/*background: black;*/
	selection-color: rgb(216, 216, 216);
  	background-color: rgb(30, 30, 30);
}
</string>
  </property>
  <widget class="QWidget" name="Titlebar_widget" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>781</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(27, 27, 27);</string>
   </property>
   <widget class="QLabel" name="Titlebar_label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>0</y>
      <width>251</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Entry Editor</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignmentFlag::AlignBottom|Qt::AlignmentFlag::AlignLeading|Qt::AlignmentFlag::AlignLeft</set>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="Back_button">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>380</y>
     <width>141</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>  Back     </string>
   </property>
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/icons/white-left-arrow.png</normaloff>
     <disabledoff>:/icons/icons/white-left-arrow-disabled.png</disabledoff>:/icons/icons/white-left-arrow.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>15</width>
     <height>15</height>
    </size>
   </property>
  </widget>
  <widget class="QLineEdit" name="Name_textedit">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>110</y>
     <width>481</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit
{
	border: 1px solid rgb(30, 30, 30);
	background-color: black;
	border-radius: 10px;
	padding-left: 10px;
	color: rgb(20, 205, 255); 
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="frame">
    <bool>false</bool>
   </property>
   <property name="placeholderText">
    <string>Name your entry</string>
   </property>
  </widget>
  <widget class="QLabel" name="EntryName_label">
   <property name="geometry">
    <rect>
     <x>11</x>
     <y>120</y>
     <width>131</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>PlayList Name:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignmentFlag::AlignRight|Qt::AlignmentFlag::AlignTrailing|Qt::AlignmentFlag::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="Save_button">
   <property name="geometry">
    <rect>
     <x>630</x>
     <y>380</y>
     <width>141</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>  Save</string>
   </property>
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/icons/white-diskette.png</normaloff>
     <disabledoff>:/icons/icons/white-diskette-disabled.png</disabledoff>:/icons/icons/white-diskette.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>20</width>
     <height>20</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="OpenFiles_button">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>230</y>
     <width>35</width>
     <height>51</height>
    </rect>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius: 10px;
/*background-color: rgba(1, 0, 0, 0);*/
</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/icons/white-folder.png</normaloff>
     <disabledoff>:/icons/icons/white-folder-disabled.png</disabledoff>:/icons/icons/white-folder.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>20</width>
     <height>20</height>
    </size>
   </property>
   <property name="flat">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="Source_label">
   <property name="geometry">
    <rect>
     <x>11</x>
     <y>230</y>
     <width>131</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Source:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignmentFlag::AlignRight|Qt::AlignmentFlag::AlignTrailing|Qt::AlignmentFlag::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QTextEdit" name="Source_textedit">
   <property name="geometry">
    <rect>
     <x>151</x>
     <y>230</y>
     <width>481</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius: 10px;
color: rgb(20, 205, 255); 
padding-left: 40px;</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::Shape::NoFrame</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Shadow::Plain</enum>
   </property>
   <property name="lineWrapColumnOrWidth">
    <number>0</number>
   </property>
   <property name="acceptRichText">
    <bool>false</bool>
   </property>
   <property name="placeholderText">
    <string>Paste full link here (https://myplaylistlink.com/list) or select a File in Dialog</string>
   </property>
  </widget>
  <widget class="QComboBox" name="SourceType_combobox">
   <property name="geometry">
    <rect>
     <x>151</x>
     <y>170</y>
     <width>120</width>
     <height>30</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(20, 205, 255); 
</string>
   </property>
   <item>
    <property name="text">
     <string>file</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>url</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="SourceType_label">
   <property name="geometry">
    <rect>
     <x>11</x>
     <y>180</y>
     <width>131</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Source Type:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignmentFlag::AlignRight|Qt::AlignmentFlag::AlignTrailing|Qt::AlignmentFlag::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QWidget" name="background_widget" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>30</y>
     <width>801</width>
     <height>421</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: qlineargradient(spread:pad, x1:0.369, y1:0.182, x2:1, y2:1, stop:0 rgba(57, 9, 126, 255), stop:1 rgba(129, 18, 129, 255));</string>
   </property>
  </widget>
  <zorder>background_widget</zorder>
  <zorder>Titlebar_widget</zorder>
  <zorder>Back_button</zorder>
  <zorder>Name_textedit</zorder>
  <zorder>EntryName_label</zorder>
  <zorder>Save_button</zorder>
  <zorder>Source_label</zorder>
  <zorder>SourceType_combobox</zorder>
  <zorder>SourceType_label</zorder>
  <zorder>Source_textedit</zorder>
  <zorder>OpenFiles_button</zorder>
 </widget>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
